### DISCUSSION TITLE
set condition for filament admin panel authentication

### DISCUSSION THREAD

**Discussion by amirmmdz** [filamentphp/filament#12581] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hi
I have a field with name type in my User model and i need to set a condition that only users can login to admin panel that their type field is greather that 1

**Comment by declum** [comment]
> Hi
> 
> I have a field with name type in my User model and i need to set a condition that only users can login to admin panel that their type field is greather that 1


Did you try canAccessFilament with checks fhat uses the field you need?

https://filamentphp.com/docs/2.x/admin/users

**Comment by maartenpaauw** [comment]
@declum answer is the right solution. Here is a link to the v3 docs: https://filamentphp.com/docs/3.x/panels/installation#allowing-users-to-access-a-panel
